src/deterministic_vi/elbo_objective.jl

```diff
@@ -36,39 +36,54 @@ function calculate_source_pixel_brightness!{NumType <: Number}(
     @inbounds for i in 1:Ia # Stars and galaxies
         fsm_i = (i == 1) ? fs0m : fs1m
         a_i = ea.vp[s][ids.a[i, 1]]
+        sb_E_l_a_b_i = sb.E_l_a[b, i]
```

Comment:
Yeah, I did most of them in one fell swoop so there might be some inconsequential ones in there, especially since some of these loads are via fields of an immutable. If you find ones that don't matter, let me know and I'll revert them.